### PR TITLE
Add username labels to map/minimap

### PR DESCRIPTION
**Pull request by Maxattax97**

### CODE REVIEW

**Comment by Anuken** [comment]
How does this look on the small minimap, or in situations with more than a few players? I'm worried that in large servers, this would clog up the map and make things hard to see.

File: core/src/io/anuke/mindustry/graphics/MinimapRenderer.java [hunk 1]
```diff
@@ -82,6 +86,15 @@ public void drawEntities(float x, float y, float w, float h){
         for(Unit unit : units){
             float rx = (unit.x - rect.x) / rect.width * w, ry = (unit.y - rect.y) / rect.width * h;
             Draw.color(unit.getTeam().color);
+
+            if (unit.getTypeID() == TypeIDs.player) {
```
Comment (Anuken) [inline core/src/io/anuke/mindustry/graphics/MinimapRenderer.java]: This is a strange way to compare types. Use `unit instanceof Player` instead.

**Comment by Maxattax97** [comment]
I took out the local player's name from displaying originally because it didn't seem useful, but I kept the others' names on the minimap because I thought it might be okay. I've taken those out, but kept them on the large map dialog.

I was playing the other night with my friends and one would say "come here, look at this" but in order to figure out _where_ they were, geographical descriptions had to be given. I felt I'd add this feature to alleviate that.

Here are some new screenshots:

File: core/src/io/anuke/mindustry/graphics/MinimapRenderer.java [hunk 2]
```diff
@@ -80,14 +84,28 @@ public void drawEntities(float x, float y, float w, float h){
         rect.set((dx - sz) * tilesize, (dy - sz) * tilesize, sz * 2 * tilesize, sz * 2 * tilesize);
 
         for(Unit unit : units){
-            float rx = (unit.x - rect.x) / rect.width * w, ry = (unit.y - rect.y) / rect.width * h;
+            float rx = (unit.x - rect.x) / rect.width * w;
+            float ry = (unit.y - rect.y) / rect.width * h;
+
+            if (withLabels && unit instanceof Player) {
```
Comment (Anuken) [inline core/src/io/anuke/mindustry/graphics/MinimapRenderer.java]: Use compact formatting (e.g. `if(){`). Happens a few times in this code.

**Comment by Anuken** [comment]
Looks good, thanks.

**Pull request merged**